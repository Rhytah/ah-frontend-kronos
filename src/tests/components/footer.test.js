import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';

import Footer from '../../components/common/Footer/Footer';

describe('<Footer />', () => {
  const wrapper = shallow(<Footer />);

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
