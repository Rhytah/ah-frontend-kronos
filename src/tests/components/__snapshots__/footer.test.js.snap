// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
<Fragment>
  <div
    className="footer-dark"
  >
    <footer>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-md-3 item text"
          >
            <ul>
              <li>
                <a
                  className="a-text"
                  href="#"
                >
                  About
                </a>
              </li>
              <li>
                <a
                  className="a-text"
                  href="#"
                >
                  Authors
                </a>
              </li>
              <li>
                <a
                  className="a-text"
                  href="#"
                >
                  Contact
                </a>
              </li>
            </ul>
          </div>
          <div
            className="col-md-6 item text"
          >
            <h3>
              Authors-haven
            </h3>
            <p>
              Praesent sed lobortis mi. Suspendisse vel placerat ligula. Vivamus ac sem lacus. Ut vehicula rhoncus elementum. Etiam quis tristique lectus. Aliquam in arcu eget velit pulvinar dictum vel in justo.
            </p>
          </div>
          <div
            className="col-md-3 item social"
          >
            <a
              href="#"
            >
              <i
                className="fab fa-facebook-square social-fab"
              />
            </a>
            <a
              href="#"
            >
              <i
                className="fab fa-twitter social-fab"
              />
            </a>
            <a
              href="#"
            >
              <i
                className="fab fa-instagram social-fab"
              />
            </a>
          </div>
        </div>
      </div>
    </footer>
  </div>
</Fragment>
`;
